consul/src/main/scala/io/buoyant/consul/v1/ConsulApi.scala

```diff
@@ -161,5 +175,6 @@ case class ServiceNode(
   ServiceName: Option[String],
   ServiceTags: Option[Seq[String]],
   ServiceAddress: Option[String],
-  ServicePort: Option[Int]
+  ServicePort: Option[Int],
+  Status: Option[HealthStatus.Value]
```

Comment:
It looks like this addition is causing compile errors in `ConsulNamerTest` (which is what's currently breaking [the CI build](https://circleci.com/gh/linkerd/linkerd/5794)), since the test creates some `ServiceNode`s without passing an argument for the `Status` field. 

We'll want to add the `Status` field to the `ServiceNode`s here 
https://github.com/linkerd/linkerd/blob/ef38c6ad20071a57294bae863923aae99572b5dd/namer/consul/src/test/scala/io/buoyant/namer/consul/ConsulNamerTest.scala#L15-L32
so the test can compile.

Reply:
Fixed and passing